app/src/main/res/values/styles.xml

```diff
@@ -13,4 +13,11 @@
     <style name="MyKeyboardViewPopup" parent="MyKeyboard">
         <item name="keyTextSize">@dimen/preview_text_size</item>
     </style>
+
+    <style name="MyKeyboard.Alert" parent="Theme.Material3.Light.Dialog.Alert">
```

Comment:
is this used anywhere?

Reply:
Yes, it's used  [here](https://github.com/KryptKode/Simple-Keyboard/blob/feat/switch-language-on-longpress-emoji/app/src/main/kotlin/com/simplemobiletools/keyboard/extensions/Context.kt#L51) and [here](https://github.com/KryptKode/Simple-Keyboard/blob/feat/switch-language-on-longpress-emoji/app/src/main/kotlin/com/simplemobiletools/keyboard/extensions/Context.kt#L53) because we are showing the dialog from a service without a theme, we need to define the  style. `Dialog.Alert` defines the min width of the dialog

Reply:
I could not use the one from commons (the [MyDialogTheme](https://github.com/SimpleMobileTools/Simple-Commons/blob/master/commons/src/main/res/values/styles.xml#L179)) because as it doesn't have the min-width for Alert dialog defined. It looked weird 

![image](https://user-images.githubusercontent.com/25648077/200352578-85d028f8-f032-4d1a-bd4d-208c07f9bb7b.png)